app/viewmodels/cya/CYAEntryDetailsSummaryListHelper.scala

```diff
@@ -108,22 +108,11 @@ trait CYAEntryDetailsSummaryListHelper {
 
   private def buildEntryDateListRow(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
     answers.get(EntryDetailsPage).map { entryDetails =>
-      val entryMonth = entryDetails.entryDate.getMonthValue.toString match {
-        case "1"  => messages("month.1")
-        case "2"  => messages("month.2")
-        case "3"  => messages("month.3")
-        case "4"  => messages("month.4")
-        case "5"  => messages("month.5")
-        case "6"  => messages("month.6")
-        case "7"  => messages("month.7")
-        case "8"  => messages("month.8")
-        case "9"  => messages("month.9")
-        case "10" => messages("month.10")
-        case "11" => messages("month.11")
-        case "12" => messages("month.12")
-      }
+      val entryMonth = entryDetails.entryDate.getMonthValue
```

Comment:
you don't need to create this temporary variable just use
messages(s"month.${entryDetails.entryDate.getMonthValue}") when building the entryDate

strictly speaking entryDateFormat isn't accurate as the variable contains the entryDate not the Format of it.